src/lib.rs

```diff
@@ -593,6 +620,18 @@ impl<S: SuffixScheme> Write for FileRotate<S> {
                     file.write_all(buf)?;
                 }
             }
+            ContentLimit::Date => {
+                let local: DateTime<Local> = Local::now();
+                if let Some(modified) = self.modified {
```

Comment:
Is it generally the assumption that if we have `ContentLimit::Date`, then `self.modified` is Some?